src/client/datascience/notebook/cssStyles/tableStyles.js

```diff
@@ -5,6 +5,6 @@ tableStyleSheet.innerHTML = `#container > div > div:not(.preview) > div {overflo
 document.body.appendChild(tableStyleSheet);
 var dataframes = document.getElementsByClassName('dataframe');
 for(i=0;i<dataframes.length;i++) {
-    dataframes[0].parentElement.style.width = '100%';
-    dataframes[0].parentElement.style.overflowX = 'auto';
+    dataframes[i].parentElement.style.width = '100%';
```

Comment:
Yup :(. Can't believe I did that for a point release. Was doing all the testing with a file with a single dataframe in a test file, never even realized that I moved over the dev console code without swapping that.